### PR TITLE
Dialog

### PR DESCRIPTION
**Pull request by jonmattgray**
https://user-images.githubusercontent.com/40722025/211337818-863af4bd-a3c1-4091-9b92-d16b786fb9f2.mp4

- Opens card on list or map point click. List item is highlighted and auto-scrolled to.
- Closing dialog unhighlights. Dialog closes on search.
- Also adds a mapbox style that is an exact copy of the style used to make the designs

### CODE REVIEW

**Review by dblane-digicatapult — DISMISSED** [review]

**Review by dblane-digicatapult — DISMISSED** [review]

File: src/pages/Home.js [hunk 1]
```diff
@@ -146,23 +162,31 @@ export default function Home() {
       </Grid.Panel>
       <Grid.Panel area="projects">
         <ListWrapper>
-          {filteredGeoJson.features.map((i, index) => (
+          {filteredGeoJson.features.map((feature, index) => (
             <ListCard
-              key={index} //TODO assign ID?
-              title={`${i.properties['Name']}`}
-              subtitle={`${i.properties['Name of Lead Partner']}`}
+              key={index}
```
Comment (mattdean-digicatapult) [inline src/pages/Home.js]: should this be off the new ID?

**Comment by mattdean-digicatapult** [comment]
What's going on with the weird borders (in the dialog)?

**Comment by jonmattgray** [comment]
> What's going on with the weird borders (in the dialog)?

Hard-coded in the component, could change it to `overflow: auto`? https://github.com/digicatapult/ui-component-library/blob/f4d7f0b95e7be3a4cf459e92845f7d67ca335551/src/Dialog/index.tsx#L77

File: src/pages/Home.js (hunk 1 above)
Comment (jonmattgray) [reply src/pages/Home.js]: changed

**Comment by mattdean-digicatapult** [comment]
> > What's going on with the weird borders (in the dialog)?
> 
> Hard-coded in the component, could change it to `overflow: auto`? https://github.com/digicatapult/ui-component-library/blob/f4d7f0b95e7be3a4cf459e92845f7d67ca335551/src/Dialog/index.tsx#L77

I'm wondering if the way to handle it is getting rid of `width: 100%` on the form?

**Comment by mattdean-digicatapult** [comment]
> > What's going on with the weird borders (in the dialog)?
> 
> Hard-coded in the component, could change it to `overflow: auto`? https://github.com/digicatapult/ui-component-library/blob/f4d7f0b95e7be3a4cf459e92845f7d67ca335551/src/Dialog/index.tsx#L77

And why don't we see this in the storybook HII story for the dialog?

**Review by n3op2 — COMMENTED** [review]
as per matts comment, yea, wonder what is wrong with the borders, looks like it's trying to render scroll bar, can we set scroll; none? i mean have we tried?

sorry for double was not able to reply to @mattdean-digicatapult comment

**Comment by jonmattgray** [comment]
Converting to draft until a couple of `ui-component-library` changes are made

**Review by mattdean-digicatapult — APPROVED** [review]

**Pull request merged**